modules/meteroid/src/bin/seeder.rs

```diff
@@ -25,6 +27,7 @@ async fn main() -> error_stack::Result<(), SeederError> {
     let store = Store::new(
         env::var("DATABASE_URL").change_context(SeederError::InitializationError)?,
         crypt_key,
+        Arc::new(eventbus::memory::InMemory::new()),
```

Comment:
hm, so here we create an eventbus but do not attach any handlers to it? 
@gaspb I think for seeder we can use a noop eventbus?